desc/experimental/_terpsichore.py

```diff
@@ -0,0 +1,872 @@
+            nj, nk, ivac, lssl, lssd
+        )
+    )
+    f.write("C\n")
+    f.write("C     TABLE OF FOURIER COEFFIENTS FOR BOOZER COORDINATES\n")
+    f.write("C     EQUILIBRIUM SETTINGS ARE COMPUTED FROM FIT/VMEC\n")
+    f.write("C\n")
+
+    boz_str_title = "C M=  0"
+    boz_str_neg = "      0"
+    boz_str_pos = "      1"
+    for m in range(1, 37):
+        if m >= 10:
+            boz_str_title += " " + str(m)[1]
+        else:
+            boz_str_title += " " + str(m)
+        boz_str_neg += " 1"
+        boz_str_pos += " 1"
+    boz_str_title += "  N\n"
+    f.write(boz_str_title)
+
+    for n in range(-N_booz_max, N_booz_max + 1):
+        if n < 0:
+            f.write(boz_str_neg + "{:>3}\n".format(n))
+        else:
+            f.write(boz_str_pos + "{:>3}\n".format(n))
+
+    f.write("C\n")
+    f.write("      LLAMPR      LVMTPR      LMETPR      LFOUPR\n")
+    f.write("           0           0           0           0\n")
+    f.write("      LLHSPR      LRHSPR      LEIGPR      LEFCPR\n")
+    f.write("           9           9           1           1\n")
+    f.write("      LXYZPR      LIOTPL      LDW2PL      LEFCPL\n")
+    f.write("           0           1           1           1\n")
+    f.write("      LCURRF      LMESHP      LMESHV      LITERS\n")
+    f.write("           1           1           2           1\n")
+    f.write("      LXYZPL      LEFPLS      LEQVPL      LPRESS\n")
+    f.write("           1           1           0           0\n")
+    f.write("C\n")
+    f.write(
+        "C    PVAC        PARFAC      QONAX        QN         DSVAC       "
+        + "QVAC    NOWALL\n"
+    )
+    f.write(
+        "  1.0001e+00  0.0000e-00  0.6500e-00  0.0000e-00  1.0000e-00  "
+        + "1.0001e+00     -2\n"
+    )
+    f.write("\n")
+    f.write(
+        "C    AWALL       EWALL       DWALL       GWALL       DRWAL       DZWAL   "
+        + "NPWALL\n"
+    )
+    f.write(
+        "  {:10.4e}  ".format(awall)
+        + "1.5000e+00 -1.0000e-00  5.2000e-00 -0.0000e-00 +0.0000e-00      2\n"
+    )
+    f.write("C\n")
+    f.write("C    RPLMIN       XPLO      DELTAJP       WCT        CURFAC\n")
+    f.write(
+        "  1.0000e-05  1.0000e-06  {:10.4e}  6.6667e-01  1.0000e-00\n".format(deltaJp)
+    )
+    f.write("C\n")
+    f.write(
+        "C                                                             "
+        + "MODELK =      {}\n".format(modelk)
+    )
+    f.write("C\n")
+    f.write(
+        "C     NUMBER OF EQUILIBRIUM FIELD PERIODS PER STABILITY PERIOD: "
+        + "NSTA =      {}\n".format(nfp)
+    )
+    f.write("C\n")
+    f.write("C     TABLE OF FOURIER COEFFIENTS FOR STABILITY DISPLACEMENTS\n")
+    f.write("C\n")
+    f.write(
+        "C M=  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 "
+        + "0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5  N\n"
+    )
+
+    for n in range(N_min, N_max + 1):
+        mode_str = "     "
+        for m in range(56):
```

Comment:
56 is what?

Reply:
should it be an input for this to change like a resolution?

Reply:
This is the max poloidal resolution that TERPSICHORE supports. It's hard-coded here to create the full table of all possible poloidal modes, but the input `M_max` determines which of those modes actually get used.

Reply:
In other words, `M_max` cannot be greater than 56